backend/api/office_hours/oh_ticket.py

```diff
@@ -0,0 +1,123 @@
+"""OH Ticket API
+
+This API is used to access OH ticket data for history purposes."""
+
+from fastapi import APIRouter, Depends
+
+from backend.models.office_hours.oh_ticket import (
+    OfficeHoursTicket,
+    OfficeHoursTicketDraft,
+)
+from backend.models.office_hours.oh_ticket_details import OfficeHoursTicketDetails
+from ..authentication import registered_user
+from ...models import User
+
+
+__authors__ = ["Sadie Amato", "Bailey DeSouza"]
+__copyright__ = "Copyright 2024"
+__license__ = "MIT"
+
+
+api = APIRouter(prefix="/api/office-hours/ticket")
+openapi_tags = {
+    "name": "Office Hours",
+    "description": "Office hours ticket functionality",
+}
+
+
```

Comment:
Do you guys think we would also need an API route to get a ticket by `id`? I guess the post and update returns the Ticket Details model, but it may help to have this endpoint still? If not explicitly used on the front end, it could still be a helpful endpoint for development purposes. Let me know what you guys think!

Reply:
Added this in my most recent commit! Yes, i think it will be helpful for us to have even if there's not a use case in actual usage.